README.md

```diff
@@ -1,23 +1,51 @@
 <img src="http://moveit.ros.org/assets/images/moveit2_logo_black.png" alt="MoveIt! Logo" width="200"/>
 
------
+The MoveIt Motion Planning Framework for ROS 2
 
-Contributions are being reviewed. Refer to https://github.com/acutronicrobotics/moveit2 for a fork maintained by @AcutronicRobotics that includes the latest changes.
+We are currently working on the upcoming Beta version of MoveIt 2.
+More information about our roadmap and future milestones will be added soon.
 
------
+### Build from Source
+
+> Note: Currently, only the moveit\_core packages are being compiled.
```

Comment:
Is this still true? The alpha release last May advertised more packages than just moveit_core

Reply:
The alpha release was based on Acutronic's fork. To my knowledge ros-planning/moveit2 has never been that far.